Update isotope intensity estimation Rmd

jorainer opened the pull request:
- restructure the document.
- reformulate definitions and part of the text.
- changes to the R code.

jorainer committed:
Merge pull request #70 from EuracBiomedicalResearch/andrea

New approach for bounds on R

jorainer committed:
Restructure isotope intensity estimation Rmd

- Restructure the isotope intensity estimation Rmd, reformulate and
  update/change parts of the code.

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -132,38 +132,55 @@ compound.
+
+Finally the probability of observing the isotopologue $I$ (and the corresponding
+mass difference $md$) is the product of the probabilities of observing each
+isotope variant related to each element.
```
Maybe, before you referred to "isotope variant" as "combination of isotopes of each element".

jorainer replied:
updated.

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -132,38 +132,55 @@ compound.
 
 $$P(I) = \prod P(I_{i})$$
 
+There is a very high number $N_{I_X} = \binom{q + n_X -1}{n_X}$ of possible
```
I think that q is no longer defined. So maybe if we keep this formula we should say that it is the number of different isotopes that element X has

jorainer replied:
thanks! I will add it.

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -132,38 +132,55 @@ compound.
+is higher than a pre-defined proportion of the monoisotopic peak.
+
 Finally, the ratio between the intensities of the monoisotopic peak $x_0$ and a
 peak of an isotopologue $x_I$ is supposed to be equivalent to the ratio of their
```
Just as a note, I think for a real spectrum the ratios of intensities approximate the ratios of probabilities so maybe it's not an exact equality

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -172,32 +189,38 @@ difference $md$ of each of its isotopologues could be determined.
+
+In the present approach we first determine for each compound in HMDB all
+isotopologues that would result in a peak with an intensity (relative to the
+monoisotopic peak) above a certain threshold. We next identify the most
```
Having chosen `rel_to = 2` in `enviPat::isopattern` here should be "a peak with absolute probability above ..."

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -172,32 +189,38 @@ difference $md$ of each of its isotopologues could be determined.
+these isotopologues we then determine their mass difference and evaluate the
+observed intensity (probability) ratios between the monoisotopic peak and the
+isotopologue of all compounds in HMDB for which the isotopologue was
+reported. The mass difference is constant for the isotopologues but their
```
This sentence confuses me a bit. Each isotopologue of a given compound has a different mass, doesn't it? Or do you mean that each isotopologue with a certain substitution has the same mass?

jorainer replied:
I mean that the mass difference (isotologue peak to monoisotopic peak) is independent of the compound. in other words, [13]C isotopologues for all compounds will have the same *md*. I'll try to reformulate.

jorainer replied:
I changed this with "While the mass difference $md$ of an isotopologue peak depends only on the isotopes of its isotopic substitution, its intensity ratio depends also on the total number of atoms for the elements of the isotopic substitution that are present in the compound." - hope that's clearer.

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -172,32 +189,38 @@ difference $md$ of each of its isotopologues could be determined.
+isotopologue of all compounds in HMDB for which the isotopologue was
+reported. The mass difference is constant for the isotopologues but their
+intensity ratio depends on the total number of atoms part of the isotopic
+substitution that are present in the compound. For each isotopologue we have
```
"For each substitution" instead of "For each isotopologue"?

jorainer replied:
I would like to keep "isotopologue" here - analytical chemist will understand that bettern than "substitution".

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -172,32 +189,38 @@ difference $md$ of each of its isotopologues could be determined.
+
+## Properties
+
+- This approach considers isotopologues, i.e. all isotopic substitutions
```
So we can use the terms "substitution" and "isotopologue" interchangeably? I thought a substitution represented the difference of an isotopologue with respect to the monoisotopic version but not the isotopologue itself. Sorry, I'm probably too nitpicking, but just to know :)

jorainer replied:
Aaaah. I got confused. Let's call it "isotopologue" for now. Then we let Vinicius read through the final document to comment on this.

andreavicini commented on isotope-intensity-estimation.Rmd:
````diff
@@ -553,153 +495,194 @@ peak for at least a half of the compounds in which they are possible.
+```
+
+For each isotopologue we have thus defined the observed intensity (probability)
+ratio for compounds in HMDB. The probability of an isotope for a single element
````
Here I think it should be the "combination of isotopes" instead of "isotope"?

andreavicini commented on isotope-intensity-estimation.Rmd:
```diff
@@ -553,153 +495,194 @@ peak for at least a half of the compounds in which they are possible.
+For some elements clearly a linear relationship between the element count and
+the compound's mass is visible. The ratio of the probabilities between the
+isotopologue and its monoisotopic form depends only on the probabilities of the
+isotopes forming the substitution. These probabilities are products on the
```
again, I would say not of the "isotopes" but of what was called "combination of isotopes of a given element"?

andreavicini reviewed (approved):
Thanks @jorainer for the nice updates! I only added a few comments/ questions related to some descriptions. I'll now merge the PR and do changes later if you think they are appropriate

jorainer commented:
Thanks @andreavicini for the review. I will make the changes you suggest and update the pull request.

jorainer committed:
Address Andrea's comments

jorainer commented:
I've addressed all your comments @andreavicini - with the exception if we should call the "thing" "isotopologue" or "isotopic substitution" - I would let Vinicius read the final rendered document and then we have a short meeting discussing how to best (correctly) name it.
